src/gleam_stdlib.erl

```diff
@@ -203,6 +204,9 @@ string_pop_grapheme(String) ->
         _ -> {error, nil}
     end.
 
+string_pop_codeunit(<<Cp/integer, Rest/binary>>) -> {Cp, Rest};
```

Comment:
This doesn't pop a codepoint, it pops a byte. Are we sure this is doing the right thing?

Also, is this used at all?

Reply:
It pops a _code unit_, which means a unit of storage based on the encoding. So it pops a byte on Erlang because strings are UTF-8 encoded, or 16 bit (one "index increment") on Javascript.

Reply:
Codepoints are between 1 and 4 bytes in size on utf8.

Reply:
Yes, which is why these functions work on _code units_ instead to the byte indices are correct and we can slice in constant time.